teps/0112-replace-volumes-with-workspaces.md

```diff
@@ -235,6 +247,7 @@ Cons:
 
 In this solution, if a `Task` author declares a Workspace as "internal", Tekton will provide a volume type based on the `Task` implementation.
 For `TaskRuns`, this would likely be an emptyDir volume.
+"Internal" workspaces cannot have workspace bindings specified in TaskRuns or PipelineRuns.
```

Comment:
Hmmm, if a workspace cannot have a binding, should we even call it a workspace? 🤔